api/src/main/java/org/openmrs/module/eptsreports/reporting/library/indicators/BreastfeedingIndicators.java

```diff
@@ -73,14 +73,4 @@ public CohortIndicator getBreastfeedingWomenWithViralLoadIn12MonthsAndNotDocumen
 		    txPvls.getBreastfeedingWomenAndNotDocumentedWithViralLoadResultsDenominator(),
 		    "startDate=${startDate},endDate=${endDate},location=${location}"));
 	}
-	
-	/**
-	 * Breastfeeding and pregnant women on routine
-	 * 
-	 * @return CohortIndicator
-	 */
-	public CohortIndicator getBreastfeedingAndPregnantOnRoutine() {
-		return newCohortIndicator("breastfeedingWomenWithViralLoad", EptsReportUtils.map(
```

Comment:
You'll notice that they all call `newCohortIndicator()` which means we are adding a layer which is not providing much value. 

I was thinking we actually move the `newCohortIndicator()`  calls to the dataset, which will remove the need for these files and reduce overall clutter of the module.

Reply:
Right, am reworking on those